syntax = "proto3";

package rj_Proxy;

message ProxyMessage
{
    bytes type = 1;
    bytes request_msg = 2;
}

message ProxyResponse
{
    bytes type = 1;
    bytes response_msg = 2;
}